src/ixdat/backends/directory_backend.py

```diff
@@ -27,24 +45,33 @@ class DirBackend(BackendBase):
     def __init__(
         self,
         directory=CFG.standard_data_directory,
+        project_name=CFG.default_project_name,
         metadata_suffix=CFG.standard_metadata_suffix,
         data_suffix=CFG.standard_data_suffix,
     ):
         """Initialize a directory database backend with the directory as Path
 
         Args:
-            directory (Path): the directory to save files (under table subfolders)
+            directory (Path): the main ixdat directory
+            project_name (str): the name of the project (ixdat subdirectory)
             metadata_suffix (str): The suffix to use for JSON-formatted metadata files
             data_suffix (str): The suffix to use for numpy-formatted data files
         """
-        self.directory = directory
+        self.project_directory = directory / project_name
+        if not self.project_directory.exists():
+            try:
+                self.project_directory.mkdir()
```

Comment:
I don't know if this path might reach into several layers of new directories. If so, you might want to add `parents=True`   https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir (also says something about exceptions) and then, you might add `exists=True` and remove the if.

Reply:
Thanks for the tricks 😄 . I've made the improvement here, and also in config.py.